package.json

```diff
@@ -30,6 +30,8 @@
         "deploy:moonbeam": "npx hardhat run --network moonbeam scripts/deploy.ts",
         "deploy:arbitrum": "npx hardhat run --network arbitrum scripts/deploy.ts",
         "deploy:fuse": "npx hardhat run --network fuse scripts/deploy.ts",
+        "deploy:optimism-frendLend": "npx hardhat run --network optimism scripts/deploy-frendLend.ts",
```

Comment:
not sure we need this command, also, why is it only optimism and not arbitrum as well?

Reply:
On arbitrum I used the general script and it redeployed, to the same address, the other contracts too and it cost me gas so I used the specific scripts for optimism

Reply:
I removed it now